### PR TITLE
Adds newbeedrone 2400 rx

### PR DESCRIPTION
**Pull request by Romin0615**
According to the admin, this is a new target that needs to be committed to master, so we recreated this pr, updated the lua name

### CODE REVIEW

File: src/platformio.ini [hunk 1]
```diff
@@ -18,6 +18,7 @@ extra_configs =
 	targets/namimnorc_900.ini
 	targets/namimnorc_2400.ini
 	targets/neutronrc_900.ini
+	targets/newbeedrone_2400.ini
```
Comment (pkendall64) [inline src/platformio.ini:21]: We don't actually need these changes any more.

File: src/targets/newbeedrone_2400.ini [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# ELRS-RX24D
+[env:NewBeeDrone_D_2400_RX_via_UART]
+extends = env:Unified_ESP8285_2400_RX_via_UART
+board_config = newbeedrone.rx_2400.rx24d
+
+[env:NewBeeDrone_D_2400_RX_via_BetaflightPassthrough]
+extends = env:NewBeeDrone_2400_RX_R24D_via_UART
+
+[env:NewBeeDrone_D_2400_RX_via_WIFI]
+extends = env:NewBeeDrone_2400_RX_R24D_via_UART
```
Comment (pkendall64) [inline src/targets/newbeedrone_2400.ini:14]: Not required any more as you can flash a Unified target from platformio and select the layout from the menu that is displayed at the end of the build.

**Comment by pkendall64** [comment]
> According to the admin, this is a new target that needs to be committed to master, so we recreated this pr, updated the lua name

So the other PR should be closed then?

File: src/platformio.ini (hunk 1 above)
Comment (jurgelenas) [reply src/platformio.ini:21]: We do unless unified configurator is released first before this is merged.

Comment (pkendall64) [reply src/platformio.ini:21]: We really only need an alias in Configurator because it's the same as any other generic target!

**Review by pkendall64 — CHANGES_REQUESTED** [review]
Can you remove the file and undo the change in platform.ini please, as they are unnecessary.

**Comment by brandonrc** [comment]
@pkendall64 want me to make these changes? Seems like you left this comment a month ago....

@Romin0615

**Comment by pkendall64** [comment]
> @pkendall64 want me to make these changes? Seems like you left this comment a month ago....
> 
> @Romin0615

Sure, I think I may have but the repository was not writable! I suppose you could just create a whole new PR (with fixes) and we can review and merge that.

**Comment by brandonrc** [comment]
https://github.com/ExpressLRS/ExpressLRS/pull/2179

Closing this PR and using the new PR

**Pull request closed without merging**